src/User32.Desktop/User32+ScanCode.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the Keyboard scan code constants.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// General keyboard scan code constants on the same order that it can be found on <see cref="VirtualKey" /> constants
+        /// </summary>
+        /// <remarks>Scan codes are device-dependant values, thesse are general values used by most keyboards.</remarks>
```

Comment:
BTW what is the source of this list ?

I can't find any reference to it in MSDN and there seem to be a lot of values `= 0` in there.